app/services/service_provider_updater.rb

```diff
@@ -0,0 +1,47 @@
+class ServiceProviderUpdater
+  def run
+    dashboard_service_providers.each do |sp|
+      issuer = sp['issuer'] || sp[:issuer]
```

Comment:
let's take advantage of rails's `with_indifferent_access` so we don't have to worry about both strings and sybmols?

```
sp = sp.with_indifferent_acess
```

(but do it somewhere more upstream such as in `dashboard_service_providers`